### PR TITLE
Add App Builder Authorization page

### PR DESCRIPTION
**Pull request by urseberry**
### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/service_management/app_builder/auth.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+description: Access and authentication for App builder
+---
+
+{{< callout url="https://www.datadoghq.com/dg/datadog-app-builder/" btn_hidden="false" header="Join the Beta!">}}
+Datadog App Builder is in private beta. Complete the form to request access.
+{{< /callout >}}
+
+A few tools control access and authentication for apps and their components.
+
+## App identity
```
Comment (kc-beard) [inline content/en/service_management/app_builder/auth.md:13]: we should remove this `App identity` section. it doesn't apply to apps the same way it applies to workflows. I will add clarity below but I think it's covered elsewhere, so we can just rid of this section. 

--- 

more detail: apps aren't run as the user that created them. the queries in apps are run when the app is loaded, and those queries leverage connections. users must have resolver permissions on the appropriate connections. if they don't, the queries won't run.

**Review by kc-beard — COMMENTED** [review]
left one comment but rest looks good to me, will defer to @aflinois for approval

File: content/en/service_management/app_builder/auth.md (hunk 1 above)
Comment (aflinois) [reply content/en/service_management/app_builder/auth.md:13]: Happy to remove that section

Comment (urseberry) [reply content/en/service_management/app_builder/auth.md:13]: Thank you, I'll remove it.

File: content/en/service_management/app_builder/auth.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+For more information on configuring credentials, see [Connections][2]. App Builder shares the Action Catalog and the connection credentials for each integration with [Datadog Workflow Automation][3].
+
+## App permissions
+
+### Coarse access to apps and connections
+
+Use [role-based access control (RBAC)][4] to control access to your apps and connections. 
+
+The coarse permissions that apply to apps include `apps_run` and `apps_write`
```
Comment (hestonhoffman) [inline content/en/service_management/app_builder/auth.md:29]:
```suggestion
The coarse permissions that apply to apps include `apps_run` and `apps_write`.
```

**Review by hestonhoffman — APPROVED** [review]
Tiny edit, but LGTM!

**Pull request merged**